bower.json

```diff
@@ -22,8 +22,7 @@
   "dependencies": {
     "polymer": "Polymer/polymer#0.8-preview",
     "core-selector": "Polymer/core-selector#0.8-preview",
-    "layout": "Polymer/layout",
-    "paper-styles": "PolymerLabs/paper-styles"
+    "paper-styles": "PolymerElements/paper-styles#0.8.2"
```

Comment:
Also, I'm a jerk and just pushed out `0.8.4` of paper-styles (bump polymer to `#v0.8.0-rc.4` too)